catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/data/impl/ContentItemValidator.java

```diff
@@ -26,35 +24,52 @@
  * required by the {@link ddf.catalog.content.StorageProvider}.
  */
 public class ContentItemValidator {
-
     private static final Pattern QUALIFIER_PATTERN = Pattern.compile("\\w[a-zA-Z0-9_\\-]+");
```

Comment:
Because flow control with Exceptions is gross. Also Thats what they were doing before and we might as well actually explicitly define (like a regex) what we are expecting. It is far clearer than creating a URI and doing nothing with it purely for the side effect of the exception for validation.